src/components/Carousel/index.js

```diff
@@ -75,26 +75,63 @@ export default class Carousel extends PureComponent {
     highlightOnHover: false,
     loop: false,
     overflow: false,
-    scrollCount: 1,
+    scrollCount: 2,
```

Comment:
Moving 2 at a time now?

Reply:
yes, but only by default.  this is likely to be overridden

Reply:
Heads up - I just did a quick search, but it does look like the `Carousel` is called at least one time in the main masterclass repo without specifying `scrollCount`. If we're ok with it updating to default to 2 everywhere and that behavior changing, disregard, but otherwise, that might need a find / replace before merging this in.

https://github.com/yankaindustries/masterclass/blob/staging/app/javascript/components/AllAccessPassCarousel/index.js#L30